src/Libraries/CoreNodeModels/DropDown.cs

```diff
@@ -63,13 +63,25 @@ public ObservableCollection<DynamoDropDownItem> Items
                 RaisePropertyChanged("Items");
             }
         }
+
         public override NodeInputData InputData
         {
             //TODO There is not yet an appropriate input type
```

Comment:
@SHKnudsen update this comment?

Reply:
Not entirely sure what this comment referred to with `cogs graph schema` support, so didn't want to remove anything. 

Are we okay to remove the comment?

Reply:
hmm- okay, we could change the TODO to say:
extend cogs graph schema with new `NodeInputTypes.dropdownSelection` support.